aplib.net-demo/Assets/Scripts/MainMenu/MenuButtons.cs

```diff
@@ -0,0 +1,53 @@
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class MenuPlayButton : MonoBehaviour
+{
+    public GameObject menuCanvas;
+    public GameObject settingCanvas;
+    public bool isOnSettings = false;
+
+    private void Start()
+    {
+        menuCanvas.SetActive(true);
+        isOnSettings = false;
+        settingCanvas.SetActive(false);
+    }
+
+    /// <summary>
+    /// When the play button in the main menu is clicked, it wil teleport the player to the game.
+    /// Right now it is the melee weapon scene to prove concept. THIS NEEDS TO BE CHANGED.
+    /// </summary>
+    public void PlayGame()
+    {
+        SceneManager.LoadScene("MeleeWeapon");
+    }
+
+    /// <summary>
+    /// This button toggles the menu canvas off and the setting canvas on.
+    /// </summary>
+    public void SettingToggle()
+    {
+        if (isOnSettings)
+        {
+            settingCanvas.SetActive(false);
+            isOnSettings = false;
+            menuCanvas.SetActive(true);
+        }
+        else
+        {
+            menuCanvas.SetActive(false);
+            isOnSettings = true;
+            settingCanvas.SetActive(true);
```

Comment:
Ik weet niet of hier later nog meer achterzit maar je kan het omschrijven hierheen.
```suggestion
        settingCanvas.SetActive(false);
        isOnSettings = !isOnSettings;
        menuCanvas.SetActive(true);
```

Reply:
Voor de isOnSetting is het idd handig, alleen menu en setting togglen ook. Ik heb misschien een idee hiervoor, let me cook.